src/deploy/osp_deployer/profiles/xsp.json

```diff
@@ -274,6 +282,14 @@
                   "false"
                ]
             },
+=======
```

Comment:
Resolve merge conflicts

Reply:
Don't know why it's there. Supposed to got it resolved.